model_compression_toolkit/exporter/model_wrapper/keras/builder/fully_quantized_model_builder.py

```diff
@@ -39,7 +39,7 @@ def _get_wrapper(node: common.BaseNode,
         Returns: Wrapped layer with weights quantizers and activation quantizers
 
         """
-        weights_quantizers, _ = get_quantization_quantizers(node)
+        weights_quantizers, _ = C.keras.keras_implementation.KerasImplementation().get_quantization_quantizers(node)
```

Comment:
Can you put the `get_quantization_quantizers` in the node when craeting it, so you don't need to craete the keras implementation each time?

Reply:
Not sure that I understand your suggestion, let's talk about it offline